<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="vnvpc130" name="RB_Release.Grid_EditRelease" tests="1" failures="1" timestamp="6 Jul 2012 11:51:34 GMT" time="0.000" errors="0">
  <testcase name="testGoogleSearch" time="0.000" classname="RB_Release.Grid_EditRelease">
    <failure type="java.lang.RuntimeException" message="Could not contact Selenium Server; have you started it on &#039;localhost:5554&#039; ?
Read more at http://seleniumhq.org/projects/remote-control/not-started.html
Connection refused: connect">
      <![CDATA[java.lang.RuntimeException: Could not contact Selenium Server; have you started it on 'localhost:5554' ?
Read more at http://seleniumhq.org/projects/remote-control/not-started.html
Connection refused: connect
	at com.thoughtworks.selenium.DefaultSelenium.start(DefaultSelenium.java:102)
	at RB_ReleaseArtist.Grid_CreateArtistFromAudioRelease.beforeClass(Grid_CreateArtistFromAudioRelease.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:76)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:525)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:202)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:130)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:173)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:34)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:351)
	at org.testng.internal.thread.ThreadUtil$CountDownLatchedRunnable.run(ThreadUtil.java:147)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase>
</testsuite>
